utreexo/rev.go

```diff
@@ -0,0 +1,352 @@
+package utreexo
+
+import (
+	"encoding/binary"
+	"errors"
+	"io"
+	"math"
+	"math/big"
+	"os"
+)
+
+// RevReader ...
+type RevReader struct {
+	Records []*BlockRecord
+}
+
+// BlockRecord ...
+type BlockRecord struct {
+	Magic uint32
```

Comment:
Magic should be ```[4]byte``` since it's just for checking. The rest of the repo has it as ```[4]byte``` as well.

Reply:
I also thought byte[4] was good.
However, Lit uses uint32.

https://github.com/mit-dci/lit/blob/master/wire/protocol.go#L114

Reply:
hmmm ok. @adiabat ?

I guess you can leave it alone for now